packages/flutter/test/cupertino/colors_test.dart

```diff
@@ -13,15 +11,15 @@ import '../rendering/mock_canvas.dart';
 
 class DependentWidget extends StatelessWidget {
   const DependentWidget({
-    Key key,
-    this.color,
+    Key? key,
+    required this.color,
   }) : super(key: key);
 
   final Color color;
 
   @override
   Widget build(BuildContext context) {
-    final Color resolved = CupertinoDynamicColor.resolve(color, context, nullOk: false);
+    final Color? resolved = CupertinoDynamicColor.resolve(color, context, nullOk: false);
```

Comment:
`?` and `nullOk: false` is odd. You probably just want a `!` after the resolve call.